Makefile

```diff
@@ -40,6 +42,7 @@ endif
 dependencies:
 ifeq ($(bin),main.exe)
 	@make prep-ci-local-windows
+	yarn add react-scripts@latest
```

Comment:
What's this for?

Reply:
For some wierd reason ... Even after the make Prep CI local Windows command is ran it doesn't seem to install react-script... I had to do that manually